PI/Makefile.am

```diff
@@ -28,3 +28,77 @@ src/device_state.cpp
 lib_LTLIBRARIES = libbmpi.la
 
 nobase_include_HEADERS = bm/PI/pi.h
+
+# dataplane_interface.proto
```

Comment:
I don't think this should go under the PI/ folder. It's really orthogonal.
I would be fine with a new directory at the top-level, although I am not sure about the name. Maybe "services", or "api"/"apis", with the possibility of moving other RPC service definitions there in the future.

Reply:
Thank you @antoninbas for your feedback. I've updated the PR to use "services" instead of PI/.